pycbc/workflow/grb_utils.py

```diff
@@ -539,6 +543,8 @@ def make_pygrb_plot(workflow, exec_name, out_dir,
         node.add_opt('--y-variable', tags[0])
     # Quantity to be displayed on the x-axis of the plot
     elif exec_name == 'pygrb_plot_stats_distribution':
+        seg_filelist = FileList([resolve_url_to_file(sf) for sf in seg_files])
```

Comment:
As with the other PR, we should *not* be passing around URLs (or paths) to files, but passing around `File` objects wherever possible. If seg_files is read in the top-level workflow generator, it would be better to *immediately* convert to a `File` object.